src/NzbDrone.Core/Notifications/Telegram/TelegramSettings.cs

```diff
@@ -24,7 +24,10 @@ public class TelegramSettings : IProviderConfig
         [FieldDefinition(1, Label = "Chat ID", HelpLink = "http://stackoverflow.com/a/37396871/882971", HelpText = "You must start a conversation with the bot or add it to your group to receive messages")]
         public string ChatId { get; set; }
 
-        [FieldDefinition(2, Label = "Send Silently", Type = FieldType.Checkbox, HelpText = "Sends the message silently. Users will receive a notification with no sound")]
+        [FieldDefinition(2, Label = "Topic ID", HelpLink = "https://stackoverflow.com/a/75178418", HelpText = "Specify a Topic ID to send notifications to that topic (Supergroups only)")]
+        public string TopicID { get; set; }
```

Comment:
`string` -> `int`

Reply:
Agreed that it should be an int since that's all Telegram wants anyway. However, the obstacle I'm currently facing is that I can't do the same kind of validation on an int. Any input outside of an integer gives TopicID = 0 which is the ID for the "general" topic and a "valid" input. There may be some other validation step I'm missing, but I won't get to that tonight. Or it might be best to just do no validation/check that it's not negative and just let users deal with their mistakes.

Reply:
You can do greater than or between on integers, I think there are things like Plex, kodi, etc that have ports doing that.